resources/.java.policy_move_to_home_dir

```diff
@@ -0,0 +1,84 @@
+        permission java.util.PropertyPermission "java.class.version", "read";
+        permission java.util.PropertyPermission "os.name", "read";
+        permission java.util.PropertyPermission "os.version", "read";
+        permission java.util.PropertyPermission "os.arch", "read";
+        permission java.util.PropertyPermission "file.separator", "read";
+        permission java.util.PropertyPermission "path.separator", "read";
+        permission java.util.PropertyPermission "line.separator", "read";
+
+        permission java.util.PropertyPermission "java.specification.version", "read";
+        permission java.util.PropertyPermission "java.specification.vendor", "read";
+        permission java.util.PropertyPermission "java.specification.name", "read";
+
+        permission java.util.PropertyPermission "java.vm.specification.version", "read";
+        permission java.util.PropertyPermission "java.vm.specification.vendor", "read";
+        permission java.util.PropertyPermission "java.vm.specification.name", "read";
+        permission java.util.PropertyPermission "java.vm.version", "read";
+        permission java.util.PropertyPermission "java.vm.vendor", "read";
+        permission java.util.PropertyPermission "java.vm.name", "read";
+		
+        // permissions for metafacture playground
+        
+        permission java.io.FilePermission "${user.dir}/-", "write, read, delete";
+        permission java.io.FilePermission "${java.io.tmpdir}/-", "write, read, delete";
+        permission java.io.FilePermission "${user.home}/-", "read";
+        
+        // Please adapt these pathes to all parents pathes of your user home directory
+        // Some Fix Code searches a .project file to determine an encoding
+        // It's no problem, when there's no .project file but it's a problem,
+        // when there's no access in generell to this file
+        permission java.io.FilePermission "/Users/.project", "read";
+        permission java.io.FilePermission "/.project", "read";
+        
+        permission java.util.PropertyPermission "*", "read";
+        permission java.util.PropertyPermission "jetty.git.hash", "write";
+        
+        permission java.lang.reflect.ReflectPermission "suppressAccessChecks";
+        
+        permission java.lang.RuntimePermission "getenv.*";
+        permission java.lang.RuntimePermission "createClassLoader";
+        permission java.lang.RuntimePermission "getClassLoader";
+        permission java.lang.RuntimePermission "setContextClassLoader";
+        permission java.lang.RuntimePermission "getContextClassLoader";
+        permission java.lang.RuntimePermission "accessDeclaredMembers";
+        permission java.lang.RuntimePermission "accessClassInPackage.sun.misc";
+        permission java.lang.RuntimePermission "classLoader";
+        permission java.lang.RuntimePermission "accessClassInPackage.jdk.internal.loader";
+
+        permission java.lang.management.ManagementPermission "control";
```

Comment:
Kind of surprising that this is needed (by the wormhole construct?), but seems to be less dangerous than it sounds (https://docs.oracle.com/javase/8/docs/api/java/lang/management/ManagementPermission.html), and probably nothing that could be triggered from Fix or Flux, so should be fine.